Recast/HomeViewController.swift

```diff
@@ -253,43 +289,96 @@ extension HomeViewController: UICollectionViewDataSource {
-                // swiftlint:disable:next force_cast
-                for: indexPath) as!
-                ContinueListeningCollectionViewCell
+            // swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: continueCvReuse, for: indexPath) as! ContinueListeningCollectionViewCell
             // setup CV cell
             cell.podcastImageView.image = continueDummy.image
             cell.podcastLabel.text = continueDummy.podcastTitle
             cell.titleLabel.text = continueDummy.episodeTitle
             cell.detailLabel.text = "\(continueDummy.date) · \(continueDummy.duration) min"
             cell.timeLeftLabel.text = "\(continueDummy.timeLeft) minutes left"
             return cell
+
         case .yourFavorites:
             let favoritesDummy = favoritePodcasts[indexPath.item]
-            let cell = collectionView.dequeueReusableCell(
-                withReuseIdentifier: gridCvReuse,
-                // swiftlint:disable:next force_cast
-                for: indexPath) as!
-                PodcastGridCollectionViewCell
+            // swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: gridCvReuse, for: indexPath) as! PodcastGridCollectionViewCell
             cell.seriesImageView.image = favoritesDummy.image
             cell.newStickerView.isHidden = !favoritesDummy.isNew
             return cell
+
         case .browseTopics:
             let browseDummy = browsePodcasts[indexPath.item]
-            let cell = collectionView.dequeueReusableCell(
-                withReuseIdentifier: gridCvReuse,
-                // swiftlint:disable:next force_cast
-                for: indexPath) as!
-                PodcastGridCollectionViewCell
+            // swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: gridCvReuse, for: indexPath) as! PodcastGridCollectionViewCell
             cell.seriesImageView.image = browseDummy.image
             cell.newStickerView.isHidden = !browseDummy.isNew
             return cell
         }
     }
+
+    func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
+        switch kind {
+        case UICollectionElementKindSectionHeader:
+            // swiftlint:disable:next force_cast
+            let header = collectionView.dequeueReusableSupplementaryView(ofKind: kind, withReuseIdentifier: homeHeaderReuse, for: indexPath) as! HomeCollectionViewHeader
+            if isLastSection(section: indexPath.section) {
+                header.headerTitleLabel.text = "Subscriptions"
+            } else {
+                header.headerTitleLabel.text = HomeSubsection.allCases[indexPath.section].rawValue
+            }
+            return header
+
+        default:
+            assert(false)
```

Comment:
not sure if we want an assert here bc it'll break the app, maybe return an empty `UICollectionReusableView`?